### DISCUSSION TITLE
need to submit a ROI bounding box for using NuClick model in DSA

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#1096] [Q&A]
Hi all! @SachidanandAlle @alex-virodov 
I tried using MONAILabel Annotation with NuClick within DSA in two different ways, one is drawing a bounding box (**Analysis ROI** in the left panel) and then clicking on several nuclei inside the box, the other one is clicking on several nuclei **without** drawing a bounding box. The first way works and the second way doesn't. So just want to confirm with you, the ROI is necessary for using NuClick within DSA, right?

**Comment by SachidanandAlle** [comment]
ROI is needed..
